app/components/form-input.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="field">
+  <label for={{@for}}>{{@label}}:</label>
+  <Input
+    @id={{@id}}
+    @type={{@type}}
+    @value={{@value}}
+    @class={{if @showError 'input errorBorder' 'input'}}
+    {{on 'input' (fn this.inputFieldChanged)}}
```

Comment:
Can you please tell me why fn helper is used?

Reply:
To be executed on Input change event.
Is there any convention about not using it, it would be great if you can provide a link to some blog or docs about it. 👍🏽